common/src/main/java/com/hedera/block/common/utils/FileUtilities.java

```diff
@@ -166,17 +171,38 @@ public static byte[] readGzipFileUnsafe(@NonNull final Path filePath) throws IOE
      * <p>
      * This method uses default extensions for gzip and block files.
      * <p>
-     * This method is _unsafe_ because it reads the entire file content into
-     * a single byte array, which can cause memory issues, and may fail if the
-     * file contains a large amount of data.
+     * This method is _unsafe_ because it reads the entire file content into a single byte array,
+     * which can cause memory issues, and may fail if the file contains a large amount of data.
+     *
+     * @param file to read bytes from
+     *
+     * @return byte array of the content of the file or null if the file extension is not supported
+     *
+     * @throws IOException      if unable to read the file.
+     * @throws OutOfMemoryError if a byte array large enough to contain the file contents cannot be
+     *                          allocated (either because it exceeds MAX_INT bytes or exceeds
+     *                          available heap memory).
+     */
```

Comment:
Don't reformat javadoc like this, please.

Reply:
✅ I reversed the formatting on the docs everywhere. It was indeed a problem with the IDE, I do use only spotlessApply, but looks like the IDE formatting also went through and I thought that was spotless that did that everywhere I had changes, sorry about that, I should not have this problem in the future.